create_docker_sysext.sh

```diff
@@ -3,25 +3,37 @@ set -euo pipefail
 
 ARCH="${ARCH-x86_64}"
 OS="${OS-flatcar}"
+ONLY_CONTAINERD="${ONLY_CONTAINERD-0}"
+ONLY_DOCKER="${ONLY_DOCKER-0}"
```

Comment:
I was wondering if it would make sense for possible values of these variables to be "empty" and "non-empty" instead of "1" and "anything else". You could then test them with `[[ -z … ]]` or `[[ -n … ]]` and avoid the jarring ` = 1` or `!= 1` (likely ought to be `-eq 1` and `-ne 1`). :)

But this is up to you. Not a blocker, certainly.

Reply:
Thanks, if we use `:-0` the empty case will be handled to default to `0`.